### DISCUSSION TITLE
weird score at the top in leaderboards

### DISCUSSION THREAD

**Discussion by mhdimo** [ppy/osu#19520] [Q&A] [answered]
While I was playing in lazer I found this play that had only 40 score in the leaderboard with only 1 300s

Screenshot of what I mean:
![FY_nCN9XkAIvh59](https://user-images.githubusercontent.com/62386208/182116396-45c89c29-61c4-46b4-b150-d246e4a750f0.jpg)

how is that possible? apparently the map is just one slider
also the beatmap is this one: [Dustvoxx - Trigger (Zekk Remix)](https://osu.ppy.sh/beatmapsets/626322#osu/1320129)

lazer version : 2022.731.1

**Comment by frenzibyte** [comment, marked as answer]
That player probably have edited the beatmap to one slider and submitted score. This has been fixed recently (https://github.com/ppy/osu-web/pull/9137).

**Reply by peppy** [reply]
We'll do a pass on scores like this and remove the users/scores at some point, so I wouldn't worry too much about cases like this for the time being (but reporting them is useful so we can confirm it's not an actual bug).

**Reply by Moltanicaa** [reply]
When this bug was still working, there was an oddity in the scoring system. If you activate different mods (as in the screenshot) and click on the only note on the map, you get 1.409.444 points. This number is clearly not equal to 1.000.000 + multipliers from these mods (0.06+0.12+0.06+0.12, I could be wrong). For some reason, that number came up. Perhaps this is a bug.

UPD. 99.99% = SS for some reason
![GKWYyF4qsRI.jpg](https://user-images.githubusercontent.com/59089206/182176477-6f64612c-a361-4f2d-bd70-1f263188de8e.jpg)
